README.md

```diff
@@ -1,44 +1,49 @@
 KEVM: Semantics of EVM in K
 ===========================
 
-In this repository we provide a model of the EVM in K.
+In this repository, we provide a model of the EVM in K.
 
 Fast Installation
 -----------------
 
--   `bash <(curl https://kframework.org/install)`: install [the `kup` utility](https://github.com/runtimeverification/kup)
+-   `bash <(curl https://kframework.org/install)`: install [kup package manager].
```

Comment:
Why was the link removed?

```suggestion
-   `bash <(curl https://kframework.org/install)`: install [the kup package manager](https://github.com/runtimeverification/kup).
```

Reply:
It is not removed. I moved it to the bottom of the page for consistency.